### PR TITLE
feat: add lidarr integration

### PR DESCRIPTION
**Pull request by manuel-rw**
<br/>
<div align="center">
  
  <h3>Homarr</h3>
</div>

**Thank you for your contribution. Please ensure that your pull request meets the following pull request:**

- [x] Builds without warnings or errors (``pnpm buid``, autofix with ``pnpm format:fix``)
- [x] Pull request targets ``dev`` branch
- [x] Commits follow the [conventional commits guideline](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] No shorthand variable names are used (eg. ``x``, ``y``, ``i`` or any abbrevation)

### CODE REVIEW

File: apps/nextjs/public/images/apps/imvdb.png
Comment (Meierschlumpf) [inline apps/nextjs/public/images/apps/imvdb.png:1]: Both images seem to be broken. When downloading them windows shows the following for me:
image

File: packages/integrations/src/media-organizer/lidarr/lidarr-integration.ts [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    const links: CalendarEvent["links"] = [];
+
+    if (event.artist.links.some((link) => link.name === "vgmdb")) {
+      links.push({
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+        href: event.artist.links.find((link) => link.name === "vgmdb")!.url,
+        name: "VgmDB",
+        color: "#f5c518",
+        isDark: false,
+        logo: "/images/apps/imdb.png",
```
Comment (Meierschlumpf) [inline packages/integrations/src/media-organizer/lidarr/lidarr-integration.ts:62]: Probably wrong image

**Review by Meierschlumpf — CHANGES_REQUESTED** [review]

**Review by Meierschlumpf — CHANGES_REQUESTED** [review]

File: packages/integrations/src/media-organizer/lidarr/lidarr-integration.ts [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+        mediaInformation: {
+          type: "audio",
+        },
+        links: this.getLinksForLidarrCalendarEvent(lidarrCalendarEvent),
+      };
+    });
+  }
+
+  private getLinksForLidarrCalendarEvent = (event: z.infer<typeof lidarrCalendarEventSchema>) => {
+    const links: CalendarEvent["links"] = [];
```
Comment (SeDemal) [inline packages/integrations/src/media-organizer/lidarr/lidarr-integration.ts:53]: Use a map on event.artist.links and switch statement instead? This avoids all the non-null assertions.

**Comment by SeDemal** [comment]
Also, could we switch to svg for the images since you are changing them anyway?

**Review by Meierschlumpf — APPROVED** [review]

**Pull request merged**